scripts/analysis.py

```diff
@@ -944,12 +912,14 @@ def multi_line_plot(dictionary, timestep,
         else:
             label = str(key)
 
-        plt.plot(init_year + (timestep/12),
+        plt.plot(init_year + (timestep / 12),
                  dictionary[key],
                  label=label)
         color_index += 1
-        ax = plt.gca()
-        ax.get_yaxis().set_major_formatter(plt.FuncFormatter(lambda x, loc: "{:,}".format(int(x))))
+        if sum(sum(dictionary[k]) for k in dictionary) > 1000:
```

Comment:
this line attempts to add all the values in the dictionary. so if the dictionary had two keys,
it adds (sums) the two sums of the lists. This is since
```
ax = plt.
ax.get_yaxis().set_major_formatter(
              plt.FuncFormatter(lambda x, loc: "{:,}".format(int(x))))
```
this chunk messes up the y axis if it is less than 1000 (implemented to put  commas for values greater than 1000
ie.
```
x has 2 keys, 'k1', 'k2'
the line would get sum(x['k1']) + sum(x['k2'])
```